src/main/java/com/shopping/entities/Order.java

```diff
@@ -0,0 +1,106 @@
+package com.shopping.entities;
+
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.TreeSet;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "order")
+public class Order extends BaseEntity{
+	
+	@Column
+	private LocalDate ordered;
+	
+	@Column
+	private LocalDate shipped;

```

Comment:
Diff between LocalDate and Date ?

Reply:
LocalDate has few more features than Date. LocalDate has timezones that are not supported by Date.